docs/FocusingAndIgnoring.md

```diff
@@ -100,8 +100,18 @@ describe("Ignored specs", () -> {
 ```
 #### Ignoring Specs Spectrum style
 
-Spectrum allows you to specify preconditions on a block. These preconditions can include tagging.
-The `with(BlockConfigurationChain,Block)` function is used to annotate a block with preconditions. Preconditions include:
+Spectrum allows you to specify preconditions for execution as part of the configuration of a block. These can include tagging.
```

Comment:
Removing `precondition` as a keyword in relation to the block metadata, which we're really calling `BlockConfiguration`